backend/src/nodes/utils/pil_utils.py

```diff
@@ -47,6 +49,26 @@ def resize(
     return np.array(pimg).astype("float32") / 255
 
 
+def rotate(
+    img: np.ndarray, angle: float, interpolation: int, expand: int, fill: int
```

Comment:
Please don't just use magic numbers for `expand` and `fill`. Add classes similar to `InterpolationMethod` that contain the integer constants.